tools/clang/lib/Sema/DeclSpec.cpp

```diff
@@ -309,6 +309,10 @@ bool Declarator::isDeclarationOfFunction() const {
     case TST_unspecified:
     case TST_void:
     case TST_wchar:
+    // HLSL Change Starts
+    case TST_min10float:
```

Comment:
change in behavior: These cases used to hit llvm_unreachable, but now hit false. Is this the correct behavior?

Reply:
these types can't be function declarations. So returning false seems correct.